### ISSUE TITLE
[claude-audit] serving: Architecture tab Training Loop copy stale post-#870: Huber loss + val_mae_total

### ISSUE THREAD

**Issue by alexanderdfree** [alexanderdfree/Fantasy_Football_ML_AWS#1516]
- **File**: `src/serving/frontend/src/views/Architecture.jsx:179`
- **Severity**: LOW
- **Model regress-risk**: low
- **Area**: serving
- **First seen**: @28157b8 on 2026-08-24
- **What**: The user-visible Training Loop list on the Model Architecture tab states the loss is "per-target Huber or Poisson NLL" (line 179) and that early stopping "tracks val_mae_total" (line 183). Since PR #870 every position's yards heads train on MSE, and the early-stopping monitor is the loss-weighted val MAE (`val_mae_weighted` in `src/shared/training.py`; `val_mae_total` no longer exists in src/shared). The backend /api/model_architecture training_loop block serves the same stale "per-target Huber or Poisson NLL" string — and that block is no longer consumed by the React frontend at all (Architecture.jsx reads only `data.positions` and hardcodes its own copy).
- **Why suspect**: grep of `src/*/config.py` shows MSE/poisson `head_losses` for all six positions with no huber entries; grep for `val_mae_total` in src/shared returns nothing while training.py selects on `val_mae_weighted`. The stale-Huber class was filed for docs/config comments (#1417/#1407/#1394/#1391/#1188) but not for this serving display surface — same class as the fixed #1438.
- **Suggested action**: Update the JSX bullets to per-target MSE (yards) or Poisson NLL (counts) and loss-weighted val MAE, fix the same loss string in the routes.py training_loop block, and rebuild the bundle (`npm run build`).
- **Evidence**: `<li><strong>Loss:</strong> <code>MultiTargetLoss</code> — per-target Huber or Poisson NLL (DST's four rare counts) + optional BCE on the TD gate logit.</li>`

```json
{"schema":"agent-audit/v1","audit_label":"claude-audit","provider":"Claude","file":"src/serving/frontend/src/views/Architecture.jsx","line":179,"severity":"LOW","regress_risk":"low","area":"serving","category":"unfinished_pr","first_seen_sha":"28157b8"}
```
